src/Tribe/Cache.php

```diff
@@ -180,6 +181,36 @@ public function delete_expired_transients() {
 				AND a.option_name NOT LIKE '\_transient\_timeout_tribe\_%'
 		";
 		$wpdb->query( $sql );
+
+		if ( wp_using_ext_object_cache() ) {
+			/**
+			 * Toggle trigger to control if WordPress cache should be flushed or not.
+			 *
+			 * When using real object cache we cannot selectively flush a specific cache group.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :warning: Line exceeds 80 characters; contains 95 characters

<sub>hash: d4d895f2e71eb4a97d90763c1d8e9648</sub>